### PR TITLE
add nodejs v4.3.0

### PR DESCRIPTION
**Pull request by LieAlbertTriAdrian**
## Problems ##
The new nodejs is not available in all regions (for example Singapore)

## Solutions ##
Add new nodejs image that is available in Singapore as well since we used Singapore region as well

### CODE REVIEW

File: lib/servers/eb-deployer.rb [hunk 1]
```diff
@@ -40,6 +40,7 @@
 PLATFORM = {
     "Node.js": "64bit Amazon Linux 2016.03 v2.1.3 running Node.js",
     "New Node.js": "64bit Amazon Linux 2017.03 v4.2.0 running Node.js",
+    "Node.js v4.3.0": "64bit Amazon Linux 2017.03 v4.3.0 running Node.js",
```
Comment (umarrasydan) [inline lib/servers/eb-deployer.rb]: @LieAlbertTriAdrian Node.js AMI v4.3.0

**Comment by 0505gonzalez** [comment]
@umarrasydan Don't you need node 8? Can we add node 8 onto this?

**Comment by 0505gonzalez** [comment]
LGTM

**Comment by 0505gonzalez** [comment]
Wait, @umarrasydan @LieAlbertTriAdrian will this break us-west zones?

**Comment by boxbag** [comment]
wait, why are we overwriting? We should make a new key

**Comment by umarrasydan** [comment]
@boxbag @0505gonzalez I think this solution stack doesn't exist anymore that's why it failed on creation of a new instance. But to be safe we can definitely not overwrite and add a new key.

```
[release] Running shell script
+ /tmp/jackhammer/deploy.rb servers
HEAD is now at 36fa61f Use Node v8 as ledger service
/usr/local/lib/ruby/gems/2.2.0/gems/aws-sdk-core-2.3.14/lib/seahorse/client/plugins/raise_response_errors.rb:15:in `call': No Solution Stack named '64bit Amazon Linux 2017.03 v4.2.0 running Node.js' found. (Aws::ElasticBeanstalk::Errors::InvalidParameterValue)
    from /usr/local/lib/ruby/gems/2.2.0/gems/aws-sdk-core-2.3.14/lib/aws-sdk-core/plugins/param_converter.rb:20:in `call'
    from /usr/local/lib/ruby/gems/2.2.0/gems/aws-sdk-core-2.3.14/lib/seahorse/client/plugins/response_target.rb:21:in `call'
    from /usr/local/lib/ruby/gems/2.2.0/gems/aws-sdk-core-2.3.14/lib/seahorse/client/request.rb:70:in `send_request'
    from /usr/local/lib/ruby/gems/2.2.0/gems/aws-sdk-core-2.3.14/lib/seahorse/client/base.rb:207:in `block (2 levels) in define_operation_methods'
    from /tmp/jackhammer/lib/servers/eb-deployer.rb:141:in `update_environment'
    from /tmp/jackhammer/lib/servers/eb-deployer.rb:128:in `block in deploy'
    from /tmp/jackhammer/lib/servers/eb-deployer.rb:126:in `map'
    from /tmp/jackhammer/lib/servers/eb-deployer.rb:126:in `deploy'
    from /tmp/jackhammer/lib/servers/init.rb:47:in `deploy'
    from /tmp/jackhammer/lib/servers/init.rb:55:in `run'
    from /tmp/jackhammer/deploy.rb:42:in `<main>'
```

**Comment by 0505gonzalez** [comment]
AMIs are not shared accross zones, so what works in u.s. might not work in singapore. https://github.com/xendit/xendit-ledger-service uses this image so it's definitely working in u.s.

**Comment by umarrasydan** [comment]
@0505gonzalez @boxbag updated the PR

**Comment by boxbag** [comment]
LGTM

**Comment by 0505gonzalez** [comment]
LGTM

**Pull request merged**